### DISCUSSION TITLE
strapi-plugin-rest-cache/guide/

### DISCUSSION THREAD

**Discussion by giscus** [strapi-community/strapi-plugin-rest-cache#41] [Documentation]
# strapi-plugin-rest-cache/guide/

Speed-up HTTP requests with LRU cache

https://strapi-community.github.io/strapi-plugin-rest-cache/guide/

**Comment by tsauliu** [comment]
awesome Documents dev work here, combined Vitepress with Giscus, really inspiring

**Comment by Arkaraj** [comment]
Great docs devs, just wanted to know that if this is supported for strapi 4.6, can this be used for production?

**Reply by stafyniaksacha** [reply]
Yes sure, it's already used in production on many sites!

**Comment by sshodges** [comment]
Do you need to use clearByUid if you want to manually flush a cache for a content type?

**Comment by jorge-pizzati-uvi** [comment]
Regarding the "max" option for in-memory provider, how can I set it to be 128mb?

**Comment by hfersu** [comment]
It might be a silly question, but how do I get the uid of the contentType?

**Comment by davidkcyip** [comment]
Hi, is there a way to only cache plural endpoints like /articles and not singular ones like /articles/1?

**Comment by deepaktammali** [comment]
Hi,

can we use this plugin if we are making use of the graphql endpoint?
